magento/magento2
# Frontend: Pagination overlapping when user hover last product.

### Description (*)

Frontend: Pagination overlapping when user hover last product list page.
### Fixed Issues (if relevant)

1. This is direct pr as per suggest magento reviewer.

### Manual testing scenarios (*)

1. Go to frontend > navigate nay sub category hover on last product. which is first.
2. Actual Result.
![actual-result-1](https://user-images.githubusercontent.com/43638601/51072022-458cef80-1680-11e9-99a2-d3f6b2b5187d.png)

3. Expected Result.
![expected-result](https://user-images.githubusercontent.com/43638601/51072027-576e9280-1680-11e9-951a-d30eabe2d458.png)

### Contribution checklist (*)
